src/webgpu/api/validation/encoding/encoder_state.spec.ts

```diff
@@ -78,3 +78,39 @@ g.test('pass_end_invalid_order')
       encoder.finish();
     }, true);
   });
+
+g.test('call_after_successful_finish')
+  .desc(`Test that encoding command after a successful finish generates a validation error.`)
+  .paramsSubcasesOnly(u =>
+    u.combine('passType', ['compute', 'render']).combine('IsEncoderFinished', [false, true])
+  )
+  .fn(async t => {
+    const { passType, IsEncoderFinished } = t.params;
+
+    const encoder = t.device.createCommandEncoder();
+
+    const pass = passType === 'compute' ? encoder.beginComputePass() : t.beginRenderPass(encoder);
+    pass.end();
```

Comment:
Not exactly sure what the point of this pass is, although no harm in it. Maybe it enhances the coverage slightly, so let's keep it.

Reply:
Ah, right. It was not necessary for this test. Thanks for checking it.